cookbook/python-structured-output/main.py

```
@@ -64,11 +64,11 @@ async def post(self, shared, prep_res, exec_res):
     # Read resume text from file
-    shared = {}
+    memory = {}
     with open('data.txt', 'r') as file:
         resume_text = file.read()
-    shared["resume_text"] = resume_text
+    memory.resume_text = resume_text # Use property access
 
     
     flow = Flow(start=ResumeParserNode())
-    await flow.run(shared)
+    await flow.run(memory) # Pass memory object
```

Ruff (0.8.2)

`await` statement outside of a function

(F704)